### PR TITLE
Remove from GraphQL call: Video data without renditions

### PR DESCRIPTION
**Pull request by andygout**
cc @adgad @matthew-andrews 

Fixes https://github.com/Financial-Times/next-front-page/issues/458

### CODE REVIEW

File: server/lib/backend-adapters/playlist.js [hunk 1]
```diff
@@ -5,9 +5,13 @@ class Playlist {
 
 	fetch(id, ttl = 50) {
 		return this.cache.cached(`videos.${id}`, ttl, () => {
-			return fetch(`http://next-video.ft.com/api/playlist/${encodeURI(id)}?videoFields=id,name,renditions,longDescription,publishedDate,videoStillURL`)
+			return fetch(`http:\/\/next-video.ft.com/api/playlist/${encodeURI(id)}?videoFields=id,name,renditions,longDescription,publishedDate,videoStillURL`)
```
Comment (matthew-andrews) [inline server/lib/backend-adapters/playlist.js]: are these backslashes necessary?

Comment (matthew-andrews) [inline server/lib/backend-adapters/playlist.js]: they seem to make no difference…… so safe…… but curious why :)
https://babeljs.io/repl/#?experimental=true&evaluate=true&loose=false&spec=false&code=console.log(%60http%3A%2F%2Fwww.ft.com%60)%3B%0Aconsole.log(%60http%3A%5C%2F%5C%2Fwww.ft.com%60)%3B

Comment (andygout) [inline server/lib/backend-adapters/playlist.js]: In text editor (Sublime Text) escaping the forward slashes displays remainder of line with syntax highlighting (else displays as commented-out text) and matches related parentheses. Purely for my ease when working but can remove.

**Comment by matthew-andrews** [comment]
:+1:

**Pull request merged**